cinema/views.py

```diff
@@ -43,6 +57,25 @@
 
         return MovieSerializer
 
+    def get_queryset(self):
+        queryset = self.queryset
+
+        genres = self.request.query_params.get("genres")
+        actors = self.request.query_params.get("actors")
+        title = self.request.query_params.get("title")
+
+        if genres:
+            genres = _params_to_ints(genres)
+            queryset = queryset.filter(genres__id__in=genres)
+        if actors:
+            actors = _params_to_ints(actors)
```

Comment:
In `MovieViewSet`, before calling `_params_to_ints(actors)`, ensure that `actors` is not `None` to prevent potential errors.